packages/core/test/guided-answers-api.test.ts

```diff
@@ -335,7 +340,7 @@ describe('Guided Answers Api: getNodePath()', () => {
         });
 
         // Test execution
-        const result = await getGuidedAnswerApi(options).getNodePath([111, 112]);
+        const result = await getGuidedAnswerApi(options as any).getNodePath([111, 112]);
```

Comment:
@Klaus-Keller I would need your help here. If I don't add the as any, then the tests fail with this. 

![Screenshot 2022-08-03 at 12 23 31](https://user-images.githubusercontent.com/2386570/182598300-8a075608-2958-4c49-a379-e8c19510def6.png)


Do you understand why?

Reply:
I typed the options and saw that there is a leftover in the test data: `icon`. Fixed it with commit https://github.com/SAP/guided-answers-extension/pull/198/commits/ed227e0025e829bf43632a772876a64900a1101c